src/devices/fancy-node/fancy-node.cc

```diff
@@ -27,10 +27,10 @@ static const DeviceDescription *SimpleRfBoardDescription(
 const DeviceDescription *const test_device =
     SimpleRfBoardDescription(1, {Bright});
 
-RadioHeadRadio *radio;
-NetworkManager *nm;
-FastLedManager *led_manager;
-RadioStateMachine *state_machine;
+RadioHeadRadio *radio = new RadioHeadRadio();
```

Comment:
Why not just store the object here instead of a pointer to the heap?

Reply:
(Is this related to the comment on 
https://github.com/candykingdom/firefly2/pull/51/files#diff-8c7980a9c0f5f1aafdf3541eebda36f02aed6bade3088cec21f82dec7f53bd0dR17

Reply:
Updated that other comment - node crashes if `RadioHeadRadio` is statically allocated, but I'm not sure why.